docroot/themes/humsci/humsci_basic/src/scss/_base.scss

```diff
@@ -10,10 +16,12 @@ html {
   }
 
   @include traditional {
-    color: yellow;
+    color: $su-color-cardinal-red;
   }
 
   @include themes(('traditional', 'airy')) {
     padding: 3rem;
   }
 }
+
+@import 'decanter/core/src/scss/components/brand-bar/index';
```

Comment:
do we want to keep this in here? I don't think it hurts but I thought I'd ask since I know this was also brought in just to do some testing of whether it worked